Upgrade-Guide/op_version.md

```diff
@@ -0,0 +1,29 @@
+
+###op-version
+op-version is the operating version of the Gluster which is running.
+
+After Gluster upgrade, it is important that op-version is updated in all peers to avoid any 
```

Comment:
bumping up op-version comes as part of a CLI command, mentioning "op-version is updated in all peers" could be confusing where user may feel the same command has to be run on all the nodes.